content/blog/findings-when-coverting-tests-to-pester5.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Findings when converting tests to Pester 5"
+date: 2020-05-17T05:00:00+01:00
+type: "post"
+draft: false
+author: johlju
+---
+
+If you have any questions or suggestions around this blog post then please
+reach out to `@johlju` in the Virtual PowerShell User Group [#DSC channel](https://dsccommunity.org/community/contact/).
+
+## Pester
+
+Pester is used throughout the DSC modules and their common modules to ensure
+that we maintain quality for all the contributions by both testing the new
+contribution (the tests added or changed in the contribution) and the
+new contribution together with the existing code.
+
+Pester is about to be released in a new version 5.0.0 with breaking changes
+so I set out to see what was needed to convert our existing tests that are
+run by Pester 4.x. See the release note under [Pester Releases](https://github.com/pester/Pester/releases)
+where the RC6 release notes contain documentation needed for converting.
+
+Pester is primarily maintained and developed by [Jakub Jareš](https://github.com/nohwnd).
+Pester 5 was able to be realized thanks to the hard work by Jakub during
+many weekends. If you have the means then consider [sponsoring Jakub](https://github.com/sponsors/nohwnd)
+or [sponsoring Pester](https://opencollective.com/pester) to show the
+appreciation we have for Jakub's dedication to the community to help us
```

Comment:
remove "we have".
It should be "you have" to be more natural with the beginning of the sentence, but you already used "you have the means".
So I'd probably just say: "to show your appreciation for Jakub's [...] community and improving this OSS tool." (It's not just Jakub's)